interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -398,12 +398,19 @@ public function default_action_process()
 
         $weight = $_POST["weight"];
         $height = $_POST["height"];
-        if ($weight > 0 && $height > 0) {
-            $_POST["BMI"] = ($weight / $height / $height) * 703;
+        // Validate input for weight and height
+        if (!is_numeric($weight) || $weight <= 0 || $weight > 1000 || !is_numeric($height) || $height <= 0 || $height > 120) {
+            die("Invalid weight or height. please check your input.");
```

Comment:
I like the additional checks, but we can't just die here.  User's need a way to be able to fix the problem.

Either the validation should be moved into the vitals.js file in the vitalsFormSubmitted functionality or you need to call the default_action and provide some kind of message to be passed into that function to handle the rendering.  

Let me know if this is confusing to you and I can clarify.

Reply:
Thank you for your feedback, @adunsulag. I understand the concern, and I will address it as follows:

1.  Move the validation logic to the vitalsFormSubmitted function in vitals.js to ensure user input is validated before submission.
2.  Adjust the PHP code to avoid using die() and instead call default_action with an appropriate error message for re-rendering the form.
3.  Update the template to display the error message for the user.
I'll make these changes and update the pull request shortly. Please let me know if there's anything else you'd like me to include.

Reply:
I make a pull request on this,
but in github repository all are automatically merged there is no merge and create a pull request option. can you please clarify me this issue. 
I make a pull request on PHP 8.1 - Nginx - MariaDB 11.4 on this issue also.
can you please see my pull request and any changes need.
And I have another doubt openemr organization is participated in GsoC 2025 or not.
This is my first time contribution in open source. any development needed. your advices make me more proficient in opensource contribution.
Thank you

Reply:
Your code will not be merged in automatically until it has passed a review.  I'm not sure what you are talking about things being automagically pulled.

As far as GSOC 2025, we can apply next month to be accepted as an organization.  We were not accepted in 2023 or 2024, but have been accepted at times in the past.